src/algebra/scalar.rs

```diff
@@ -155,60 +146,60 @@ impl<'de> Deserialize<'de> for Scalar {
 
 // === Addition === //
 
-/// A type alias for a result that resolves to a `Scalar`
-pub type ScalarResult = ResultHandle<Scalar>;
-/// A type alias for a result that resolves to a batch of `Scalar`s
-pub type BatchScalarResult = ResultHandle<Vec<Scalar>>;
-impl ScalarResult {
+/// A type alias for a result that resolves to a `Scalar<C>`
+pub type ScalarResult<C> = ResultHandle<C, Scalar<C>>;
```

Comment:
getting build errors here about `ResultHandle` only accepting 1 generic param, I assume that's a change that's just not in this branch?

Reply:
yeah